src/components/views/homepage/HomePurpleSection.tsx

```diff
@@ -200,7 +201,7 @@ const ForMakersContainers = styled(Col)`
 
 const ForMakersButton = styled(Button)`
 	width: 300px;
-	margin-top: 24px;
+	margin-top: auto;
```

Comment:
It's not true according to figma:

![image](https://user-images.githubusercontent.com/73441857/161967712-ed0035b1-0d3b-4c04-963b-dfc25a30ba4a.png)